internal/command/action.go

```diff
@@ -200,3 +212,26 @@ func returnStatusOfError(err error) (int, error) {
 	}
 	return 0, err
 }
+
+//Prints a list of all provider versions
+func printProviderVersions()(error){
+	providers, err := prov.GetAllProviders()
+
+	if err != nil {
+		return err
+	}
+
+	var providerVersions bytes.Buffer
+
+	for _, provider := range providers {
+		version, err := exec.Command(path.Join(prov.DefaultPath, provider), "--version").Output()
+		if err != nil {
+			providerVersions.WriteString(provider+" version uknown")
```

Comment:
`provider+" version uknown"` -> `provider + ": unknown version"`. the wording `foo version unknown` is a bit jumbly.

Reply:
The `--version` tag outputs as:
```
➜  summon git:(version-providers) ✗ summon --version            
summon version 0.8.0
```
I was following that same format

Reply:
But I do agree. I'm not sure if it's possible to change `--version` without editing the CLI library